src/util/spreadsheet/misc.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+
+def iter_commands(cr, like_all=(), like_any=()):
+    if not (bool(like_all) ^ bool(like_any)):
+        raise ValueError("Please specify `like_all` or `like_any`, not both")
+    cr.execute(
+        """
+        SELECT id,
+               commands
+          FROM spreadsheet_revision
+         WHERE commands LIKE {}(%s::text[])
+        """.format("ALL" if like_all else "ANY"),
+        [list(like_all or like_any)],
```

Comment:
The version that I proposed allowed to iterate over all the revisions by not passing `like_all` nor `like_any`.

Reply:
> The version that I proposed allowed to iterate over all the revisions by not passing like_all nor like_any.

And that can be easily added if needed... yagni?

The current implementation is to avoid accidentally iterating over all, given that no example needs that currently.

Reply:
That's a valid explanation.